assets/i18n/en.toml

```diff
@@ -80,24 +80,24 @@ vehicles = "Parking"
 noEnergy = "No meter data"
 homePower = "Consumption"
 pvProduction = "Production"
-loadpoints = "Loadpoint | Loadpoint | {count} Loadpoints"
```

Comment:
It`s "Ladepunkt" in German.
Maybe "chargepoint" would match better?

Reply:
"charging point" is too long. 
I vote for `chargepoint`

Reply:
Unfortunately not a word. Suggested "charger" below.

Reply:
Let's start with "charger". It's a bit confusing technically, since in our configuration every "loadpoint has one charger" configured to it. But I think being easier to understand is more important than being consistent with technical concepts under the hood.